lib/active_stash/validations.rb

```diff
@@ -0,0 +1,11 @@
+module ActiveStash
+  module Validations
+     class UniquenessValidator < ActiveModel::EachValidator
+    def validate_each(record, attribute, value)
+      if record.class.query(attribute => value).exists?
+        record.errors.add(attribute, options[:message] || "already exists")
+      end
+    end
+  end
+  end
```

Comment:
Looks like some wonky indenting going on there...

```suggestion
    class UniquenessValidator < ActiveModel::EachValidator
      def validate_each(record, attribute, value)
        if record.class.query(attribute => value).exists?
          record.errors.add(attribute, options[:message] || "already exists")
        end
      end
    end
  end
```

Reply:
Looks like this has been addressed.